build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -x
+# "docker manifest" requires experimental feature enabled
+export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
This probably makes it very docker specific. Does this work with `CONTAINER_CMD=podman`? If not, explain it as a requirement, and removed the `CONTAINER_CMD` indirection.

Reply:
is this is specific to docker, podman doesn't support manifest commands as of now